### ISSUE TITLE
CSS Styling

### ISSUE THREAD

**Issue by cvinczefullsail** [BradleyNatasha-Fs-1/designingforwebstandards2#7]
Organize your documents by putting your stylesheet in a *css* folder (usually named *main.css* or similar). Make sure to later update your relative URLs for your `background-image` properties.

Missing base 10 font size using 62.5% for accessibility. This will also help fix the centering issue (see CSS recreation issue)
